src/Resources/skeleton/crud/templates/index.tpl.php

```diff
@@ -19,13 +24,13 @@
                 <td>{{ <?= $helper->getEntityFieldPrintCode($entity_twig_var_singular, $field) ?> }}</td>
 <?php endforeach; ?>
                 <td>
-                    <a href="{{ path('<?= $route_name ?>_show', {'<?= $entity_identifier ?>': <?= $entity_twig_var_singular ?>.<?= $entity_identifier ?>}) }}">show</a>
-                    <a href="{{ path('<?= $route_name ?>_edit', {'<?= $entity_identifier ?>': <?= $entity_twig_var_singular ?>.<?= $entity_identifier ?>}) }}">edit</a>
+                    <a href="{{ path('<?= $route_name ?>_show', {'<?= $entity_identifier ?>': <?= $entity_twig_var_singular ?>.<?= $entity_identifier ?>}) }}">Show</a>
```

Comment:
I guess it's debatable whether or not these should be capitalized. Personally I see <th> as headings and think they should have a capital letter, but <td> as values (similar to bullet list) and don't give them a capital letter, unless it's a name or some such. 

A quick search on Google reveals there's no universal standard for table cells. In my experience many values in <td> cells are database values that could be wrongly interpreted when given a capital letter, therefore I suggest to not give them a capital letter.

Reply:
@weaverryan wdyt ?